ansible/roles/vendor/gsa.ansible-role-beats/templates/config.j2

```diff
@@ -0,0 +1 @@
+{{ package_config | to_yaml }}
```

Comment:
BTW, instead of using `template` for simple content, you can use [copy](https://docs.ansible.com/ansible/latest/modules/copy_module.html) with a `content` attribute:

```yaml
- name: copy file
  copy:
    dest: "{{ filepath}}"
    content: "{{ package_config | to_yaml }}"
```

Reply:
Will track this in the repo for the role